config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Hubstats::Engine.routes.draw do
   root to: "pull_requests#index"
 
-  post "/handler" => "events#handler", :as => "handler"
-  get "/metrics" => "repos#dashboard", :as => "metrics" 
+  post "/handler" => "events#handler", :as => :handler
+  post "/deploy" => "deploys#create"
+  get "/metrics" => "repos#dashboard", :as => :metrics 
   get "/pulls" => "pull_requests#index", :as => :pulls
+  get "/deploys" => "deploys#index", :as => :deploys
   get "/users" => "users#index", :as => :users
-  get "/repos" => "repos#index", :as => :repos
+  get "/repos" => "repos#dashboard", :as => :repos
```

Comment:
Thanks for the clarification.  It would be good to leave a short comment here noting this is here for compatibility reasons you detailed.